ui/webui/src/components/AnacondaWizard.jsx

```diff
@@ -143,6 +148,15 @@ export const AnacondaWizard = ({ dispatch, storageData, localizationData, runtim
                 ...getDiskEncryptionProps({ storageScenarioId })
             }]
         },
+        {
+            component: Accounts,
+            data: {
+                accounts,
+                setAccounts,
+                passwordPolicies: runtimeData.passwordPolicies,
+            },
+            ...getAccountsProps()
```

Comment:
We should hide the screen on WS (!isBootIso)

Reply:
Addressed in [webui: hide user screen on live images](https://github.com/rhinstaller/anaconda/pull/5280/commits/251e53528bf15e18821e282ed71674dd25cc3cae)